### DISCUSSION TITLE
Deliverables 2 Milestone

### DISCUSSION THREAD

**Discussion by meminciftci** [bounswe/bounswe2024group11#173] [Announcements]
The milestone for second deliverables is now active. For the tasks that are related to the April 30 milestone, you can connect them to it, [here](https://github.com/bounswe/bounswe2024group11/milestone/3)

@ArdaVural @ceydanursen @tenacke @hks1444 @Meminseeker @meminciftci @MucahitErdoganUnlu @ozankrkya @yunuskaydin @umitcan07

**Comment by umitcan07** [comment]
thanks
